docs/content/getting-started/code-generation.md

```diff
@@ -25,21 +24,21 @@ hof gen <cue, data, config>... [--flags...]
 
 
 There are two main modes for
-template based code generation.
+template-based code generation.
 
 1. `hof gen -T` is code gen from flags
 2. `hof gen -G` is code gen from config
 
-We will be looking at `hof gen -T` in this section.
+We will look at `hof gen -T` in this section.
 The [first example](/first-example/) is a walkthrough
 in writing a generator (the `-G` flag).
-The [getting-started/create] section will introduce
+The [getting-started/create](https://docs.hofstadter.io/getting-started/create/) section will introduce
```

Comment:
we do not want absolute links, then it doesn't work locally or on next

so just the `/getting-started/create/` in the parens

Reply:
got it, will revert! thought was missing the rest of the hyperlink 🔨